connectors/aws-dynamo-db/pom.xml

```
@@ -0,0 +1,80 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>io.camunda.connector</groupId>
+        <artifactId>connector-function-parent</artifactId>
+        <version>0.18.0-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+
+    <name>connector-aws-dynamo-db</name>
+    <description>Camunda Connector AWS DynamoDB function</description>
```

We are not using functions anymore

```suggestion
    <description>Camunda AWS DynamoDB Connector</description>
```